sites/public/src/tsx_content/professional-partners-developers-cards.tsx

```diff
@@ -0,0 +1,138 @@
+    >
+      <Card.Header>
+        <Heading priority={2} className={"text-primary-lighter font-semibold"}>
+          {t("professionalPartners.dev.whatIsTheRoleOfDevelopers")}
+        </Heading>
+      </Card.Header>
+      <Card.Section>
+        <span>
+          {t("professionalPartners.dev.whatIsTheRoleOfDevelopersResp1")}
+          <a href={`mailto:${t("professionalPartners.doorwayEmail")}`}>
+            {t("professionalPartners.dev.whatIsTheRoleOfDevelopersResp2")}
+          </a>
+        </span>
+        <DoorwaySection title={t("professionalPartners.dev.whatAreTheBenefitsOfDevelopers")}>
+          <ul className="text__medium-normal list-disc ml-5">
+            <li>{t("professionalPartners.dev.whatAreTheBenefitsOfDevelopersResp1")}</li>
+            <li>{t("professionalPartners.dev.whatAreTheBenefitsOfDevelopersResp2")}</li>
+            <li>{t("professionalPartners.dev.whatAreTheBenefitsOfDevelopersResp3")}</li>
+            <li>{t("professionalPartners.dev.whatAreTheBenefitsOfDevelopersResp4")}</li>
+            <li>{t("professionalPartners.dev.whatAreTheBenefitsOfDevelopersResp5")}</li>
+            <li>{t("professionalPartners.dev.whatAreTheBenefitsOfDevelopersResp6")}</li>
+            <li>{t("professionalPartners.dev.whatAreTheBenefitsOfDevelopersResp7")}</li>
+          </ul>
+        </DoorwaySection>
+      </Card.Section>
+    </Card>,
+    <Card
+      className="border-0"
+      key="how-does-listing-work"
+      jumplinkData={{ title: t("professionalPartners.dev.howDoesListingWork") }}
+    >
+      <Card.Header>
+        <Heading priority={2} className={"text-primary-lighter font-semibold"}>
+          {t("professionalPartners.dev.howDoesListingWork")}
+        </Heading>
+      </Card.Header>
+      <Card.Section>
+        <DoorwaySection title={t("professionalPartners.dev.whatKindOfProperties")}>
+          {t("professionalPartners.dev.whatKindOfPropertiesResp")}
+        </DoorwaySection>
+        <DoorwaySection title={t("professionalPartners.dev.howDoWeListProperty")}>
+          {t("professionalPartners.dev.howDoWeListPropertyResp1")}
+          <ul className="text__medium-normal list-disc ml-5">
+            <li>
+              {t("professionalPartners.dev.howDoWeListPropertyResp2a")}
+              <a href="https://partners.housingbayarea.mtc.ca.gov">
+                {t("professionalPartners.dev.howDoWeListPropertyResp2b")}
+              </a>
+            </li>
+            <li>
+              {t("professionalPartners.dev.howDoWeListPropertyResp3a")}
+              <a href={`mailto:${t("professionalPartners.doorwayEmail")}`}>
+                {t("professionalPartners.doorwayEmail")}
+              </a>
+              {t("professionalPartners.dev.howDoWeListPropertyResp3b")}
+            </li>
+            <li>{t("professionalPartners.dev.howDoWeListPropertyResp4")}</li>
+            <ul className="text__medium-normal list-disc ml-5">
```

Comment:
for accessibility reasons a `<ul>` can't be a child of a `<ul>`. https://dequeuniversity.com/rules/axe/4.7/list?application=axe-linter. I don't know the best way to get an indented list though with this limitation